packages/client/hmi-client/src/components/home/tera-project-card.vue

```diff
@@ -51,7 +51,7 @@
 				<section class="details">
 					<div>
 						<div class="author">{{ project?.userName ?? '——' }}</div>
-						<div class="creation-date">{{ formatDdMmmYyyy(project.createdOn) }}</div>
+						<div class="update-date">{{ formatDdMmmYyyy(project.updatedOn) }}</div>
```

Comment:
Do we even have css-classes for `creation-date` or `update-date`? If not maybe we should just remove.

Reply:
```suggestion
						<div>{{ formatDdMmmYyyy(project.updatedOn) }}</div>
```